cogs5e/models/automation/effects/damage.py

```diff
@@ -109,9 +110,14 @@ def run(self, autoctx):
         # evaluate damage
         dmgroll = d20.roll(dice_ast)
 
-        # magic arg (#853), magical effect (#1063)
+        # magic and silvered
+        always = set()
         magical_effect = autoctx.combatant and autoctx.combatant.active_effects('magical')
-        always = {'magical'} if (magical_effect or autoctx.is_spell or magic_arg) else None
+        if (magical_effect or autoctx.is_spell or magic_arg):
```

Comment:
```suggestion
        if magical_effect or autoctx.is_spell or magic_arg:
```
nit: redundant parentheses